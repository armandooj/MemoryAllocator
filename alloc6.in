a20
a30
u
q