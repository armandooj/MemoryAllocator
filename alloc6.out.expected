ALLOC at : 4 (20 byte(s))
ALLOC at : 28 (30 byte(s))
Warning, 2 blocks have not been deallocated
Warning, the dynamic allocated memory has not been completely freed
